packages/ember-glimmer/lib/component.js

```diff
@@ -430,6 +430,28 @@ export const BOUNDS = symbol('BOUNDS');
   See [Ember.Templates.helpers.yield](/api/classes/Ember.Templates.helpers.html#method_yield)
   for more information.
 
+  Layout can be used to wrap content in a component. In addition
+  to wrapping content in a Component's template, you can also use
+  the public layout API in your Component JavaScript.
+
+  ```app/components/person-profile.js
+    import Ember from 'ember';
+
+    export default Ember.Component.extend({
+      layout: Ember.HTMLBars.compile("<h1>Person's Title</h1><div class='details'>{{yield}}</div>")
```

Comment:
This is definitely correct, but more commonly these days we would use ember-htmlbars-inline-precompile babel plugin or importing an actual template module as opposed to relying on compiling the template at runtime. 

I'd suggest demonstrating the latter:

```
import layout from '../templates/components/person-profile';

export default Ember.Component.extend({
  layout
});
```